charts/incubator/pbs/values.yaml

```diff
@@ -0,0 +1,38 @@
+image:
+  repository: tccr.io/truecharts/proxmox-backup-server
+  tag: v2.2-7@sha256:dcbaee6378f32c5062518ea39a5e8e1260c3a3c609bcb32bd2dd406b7a9ef0d0
+  pullPolicy: IfNotPresent
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        targetPort: 8007
+        protocol: https
+        port: 8007
+
+persistence:
+  config:
+    enabled: true
+    mountPath: /etc/proxmox-backup
+  lib:
+    enabled: true
+    mountPath: /var/lib/proxmox-backup
```

Comment:
This will make this directory empty. Is that desired?

Reply:
I mounted this path in my tests, and now I've tested that empty works, so I added type: emptyDir to both logs and libs, and cleaned up the contents of qustions.yaml.